install_config/install/deploy_router.adoc

```diff
@@ -230,6 +230,45 @@ endif::[]
 You can link:../../admin_guide/high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 
+
+[[configuring-router-sharding]]
+=== Configuring Router Sharding
+
+Router sharding is
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels];
+you set labels on the router
+and make sure there is a corresponding node selector
+when enabling that router.
+This can be done with a single command, per shard.
+The following creates three shards, by geo.
+
+----
+$ router-shard latam
+$ router-shard emea
+$ router-shard apac
+----
```

Comment:
If this creates a router shard (I'm guessing called the `latam` or whatever), then what do you do? What does this do to your OpenShift instance? The answer to this might be better at home in the core_concepts file above. 

Also, if this section really only mentions how to create a shard, should it be called "Creating Router Shards" instead of "Configuring Router Sharding"?